java/runtime/src/main/java/io/ray/runtime/task/TaskExecutor.java

```diff
@@ -129,7 +129,13 @@ private RayFunction getRayFunction(List<String> rayFunctionInfo) {
           result = rayFunction.getConstructor().newInstance(args);
         }
       } catch (InvocationTargetException e) {
-        LOGGER.error("Execute rayFunction {} failed. actor {}, args {}", rayFunction, actor, args);
+        final boolean isIntentionalSystemExit =
+            e.getCause() != null && e.getCause() instanceof RayIntentionalSystemExitException;
+        if (!isIntentionalSystemExit) {
+          LOGGER.error(
+              "Execute rayFunction {} failed. actor {}, args {}", rayFunction, actor, args);
```

Comment:
I don't think it's reasonable to print task arguments. They may create massive outputs.

Reply:
@chaokunyang When do we need print arguments?

Reply:
@kfstorm How about printing the argument types instead?

Reply:
That sounds great!

Reply:
The number of args has a limit. In what case the output of args would be massive?

Reply:
@WangTaoTheTonic For the case that one argument with large data to be printed, instead of number of arguments.